Change monitor scenario properties from list to map (#2160)
hashgraph/hedera-mirror-node: docs/configuration.md

`hedera.mirror.monitor.nodes[].accountId`                       | ""         | The main node's account ID
`hedera.mirror.monitor.nodes[].host`                            | ""         | The main node's hostname
`hedera.mirror.monitor.nodes[].port`                            | 50211      | The main node's port
`hedera.mirror.monitor.operator.accountId`                      | ""         | Operator account ID used to pay for transactions
`hedera.mirror.monitor.operator.privateKey`                     | ""         | Operator ED25519 private key used to sign transactions in hex encoded DER format
`hedera.mirror.monitor.publish.batchDivisor`                    | 100        | The divisor used to calculate batch size when generating transactions
`hedera.mirror.monitor.publish.clients`                         | 4          | How many total SDK clients to publish transactions. Clients will be used in a round-robin fashion
`hedera.mirror.monitor.publish.enabled`                         | true       | Whether to enable transaction publishing
`hedera.mirror.monitor.publish.responseThreads`                 | 40         | How many threads to use to resolve the asynchronous responses

> `hedera.mirror.monitor.publish.scenarios.<name>`                | ""         | The unique publisher scenario name. Used as a unique identifier in logs, metrics, and the REST API
> `hedera.mirror.monitor.publish.scenarios.<name>.duration`       |            | How long this scenario should publish transactions. Leave empty for infinite

Author: Not sure if it is more clear to drop the `<name>.` and just document that `hedera.mirror.monitor.publish.scenarios` is a key value map:

```suggestion
`hedera.mirror.monitor.publish.scenarios`                | {}         | A map of scenario name to scenario properties
`hedera.mirror.monitor.publish.scenarios.duration`       |            | How long this scenario should publish transactions. Leave empty for infinite
```

Reviewer: I think for this top level calling out that it's a map is better. Otherwise the type could be confused for a string. 
You can call out that the key is a string and the value conforms to some list of properties noted below

Reviewer: Similar to how you did it for `hedera.mirror.monitor.publish.scenarios.<name>.properties`

Author: Did a hybrid approach as you suggested.
